bin/otar.ml

```diff
@@ -22,22 +22,16 @@ module Tar_gz = Tar_gz.Make
           let return x = x end)
   (struct type out_channel = Stdlib.out_channel
           type 'a io = 'a
-          let really_write oc cs =
-            let str = Cstruct.to_string cs in
+          let really_write oc str =
             output_string oc str end)
   (struct type in_channel = Stdlib.in_channel
           type 'a io = 'a
-          let really_read ic cs =
-            let len = Cstruct.length cs in
-            let buf = Bytes.create len in
-            really_input ic buf 0 len ;
-            Cstruct.blit_from_bytes buf 0 cs 0 len
-          let skip ic len = really_read ic (Cstruct.create len)
-          let read ic cs =
-            let max = Cstruct.length cs in
-            let buf = Bytes.create max in
-            let len = input ic buf 0 max in
-            Cstruct.blit_from_bytes buf 0 cs 0 len ; len end)
+          let really_read ic buf =
+            really_input ic buf 0 (Bytes.length buf)
+          let skip ic len = seek_in ic len
```

Comment:
```suggestion
          let skip ic len =
            let cur = pos_in ic in
            seek_in ic (cur + len)
```

Reply:
@dinosaure according to https://github.com/mirage/ocaml-tar/commit/37bdfe1b25d1b6d755d95dcf633ceeb9c0e9225f this is needed (absolute position for seek_in)!?

Reply:
I confirm that `seek_in` use `SEEK_SET`, so it requires an absolute offset (instead of `SEEK_CUR` which is relative).